### PR TITLE
Fix SQLite foreign on delete when altering a table

### PR DESCRIPTION
**Pull request by bittersweetshimmer**
It's just a typo fix for #4162.

### CODE REVIEW

**Comment by kibertoad** [comment]
@bittersweetshimmer Thanks! Could you also add test that was clearly missing for this case? Probably you could easily make one out of existing ones in test/integration2/schema/foreign-keys.spec.js

File: test/integration2/schema/foreign-keys.spec.js [hunk 1]
```diff
@@ -86,6 +86,58 @@ describe('Schema', () => {
...
+                  .foreign('fkey_three')
+                  .references('foreign_keys_table_three.id')
+                  .withKeyName('fk_fkey_threeee')
+                  .onDelete('CASCADE');
+              }
+            );
+
+            const queries = await builder.generateDdlCommands();
+
+            if (isSQLite(knex)) {
```
Comment (kibertoad) [inline test/integration2/schema/foreign-keys.spec.js:103]: If this test only makes sense for SQLite, can you add
`          if (!isSQLite(knex)) {
            return;
          }
`

to it?

Comment (bittersweetshimmer) [reply test/integration2/schema/foreign-keys.spec.js:103]: On it.

**Comment by kibertoad** [comment]
Thank you!

**Review by kibertoad — APPROVED** [review]

**Comment by kibertoad** [comment]
Released in 0.95.0

**Pull request merged**